packages/owl-bot/src/handlers.ts

```diff
@@ -253,42 +250,28 @@ export async function refreshConfigs(
     commitHash: commitHash,
   };
 
-  // Query github for the contents of the lock file.
-  const lockContent = await core.getFileContent(
-    githubOrg,
-    repoName,
-    owlBotLockPath,
-    commitHash,
-    octokit
-  );
-  if (lockContent) {
-    try {
-      newConfigs.lock = owlBotLockFrom(
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        yaml.load(lockContent) as Record<string, any>
-      );
-    } catch (e) {
-      logger.error(
-        `${repoFull} has an invalid ${owlBotLockPath} file: ${e.message}`
-      );
-    }
-  }
+  const response = await octokit.repos.downloadZipballArchive({
```

Comment:
I'm a little concerned this might cause some memory issues in the context of the Cloud Function, my preference would be using the `getTree` API, if possible, rather than materializing the repo on disk ([example here](https://github.com/googleapis/release-please/blob/master/src/github.ts#L1551)).

Reply:
I just tried downloading zip files for google-cloud-ruby, google-cloud-dotnet, and google-cloud-php.  google-cloud-ruby was the heaviest an 20.1 MB.  That's small enough that I'm comfortable with downloading zip balls in the cloud function.  Cloud functions are limited to 4 GB of memory.

Reply:
Also, as we discussed verbally, the getTree() call has limits.  If the limits are exceeded, then a truncated tree is returned:
https://github.community/t/github-get-tree-api-limits-and-recursivity/1300